core/common/src/AmbientOcclusion.ts

```diff
@@ -79,8 +79,20 @@ export namespace AmbientOcclusion {
         texelStepSize: this.texelStepSize !== Settings._defaultTexelStepSize ? this.texelStepSize : undefined,
         blurDelta: this.blurDelta !== Settings._defaultBlurDelta ? this.blurDelta : undefined,
         blurSigma: this.blurSigma !== Settings._defaultBlurSigma ? this.blurSigma : undefined,
+        // blurDelta: this.blurDelta,
```

Comment:
remove commented-out code?

Reply:
Yeah, caught that shortly after pushing.  Should be fixed.